src/Cards.h

```diff
@@ -0,0 +1,112 @@
+
+#pragma once
+#ifndef EIGHT_MINUTE_EMPIRE_CARDS_H
+#define EIGHT_MINUTE_EMPIRE_CARDS_H
+
+#include <string>
+#include <ostream>
+#include <vector>
+#include <ctime>
+#include <algorithm>
+#include <iostream>
+using namespace std;
+
+using std::string;
+
+class Cards {
+public:
+    Cards();
+
+    Cards(const string &good, const string &action);
```

Comment:
"good" and "action" private members should be pointers. This constructor should take in regular string type goods and actions (not pointers or references). Then in the .cpp file you assign the pointer members these passed values. Lemme know if you need help with this implementation!